src/integration/pages/frontPage/user.canonical.js

```diff
@@ -0,0 +1,13 @@
+const { canonical } = global;
```

Comment:
I made some changes so that we don't need to destructure from global so you can delete this line